xcb-proto.rb

```diff
@@ -3,35 +3,15 @@ class XcbProto < Formula
   homepage "https://www.x.org/"
   url "https://xcb.freedesktop.org/dist/xcb-proto-1.12.tar.bz2"
   sha256 "5922aba4c664ab7899a29d92ea91a87aa4c1fc7eb5ee550325c3216c480a4906"
-  revision 1
+  revision 2
   # tag "linuxbrew"
 
   bottle do
     cellar :any_skip_relocation
     sha256 "1b2b12848e39c48870a3db864572e0f4eb63fd4f9d7c715fe7765b302c378c33" => :x86_64_linux
   end
 
-  option "without-test", "Skip compile-time tests"
-  option "with-python3", "Build with python3 (default version is used otherwise)"
-
   depends_on "pkg-config" => :build
-  depends_on "libxml2" => :build if build.with? "test"
-
-  if build.with? "python3"
-    depends_on "python3" => :build
-  else
-    depends_on "python" => :build
-  end
-
-  patch :p1 do
-    url "http://www.linuxfromscratch.org/patches/blfs/svn/xcb-proto-1.12-python3-1.patch"
-    sha256 "4eef0285392c525fcb7aeb54c1b6dc8406dc33c5845c1311b65b3efc409bea2e"
-  end
-
-  patch :p1 do
-    url "http://www.linuxfromscratch.org/patches/blfs/svn/xcb-proto-1.12-schema-1.patch"
-    sha256 "3bc1a1871d17325d1f591a8ec2091f956b2071d83a6e9998d7d4880c4abf3e8b"
-  end
```

Comment:
I think this patch is still required

Reply:
Oh, well... it is needed for tests, which are deleted below, so... disregard the above comment :)

Reply:
let's maybe delete everything related to python, but leave libxml and tests

Reply:
So, we do need python. Its version does not matter but if we've got python3, then we need to apply a patch for python3.